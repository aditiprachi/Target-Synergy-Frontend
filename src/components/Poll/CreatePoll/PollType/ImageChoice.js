import styled, { css } from "styled-components";
import React from 'react'
import PropTypes from 'prop-types';
import { CardMedia, Paper, Card } from "@material-ui/core";
import Bgimage1 from '../../../images/bgimage_1.jpg'
import Bgimage2 from '../../../images/bgimage_2.jpg'
import Poll from '../../../images/polling.png'

export const Container = styled.div`
  margin: 0px auto;
  max-width: 800px;
  height: 100%;
  display: flex;
  min-height: 400px;
  flex-direction: column;
  justify-content: flex-end;
`;

export const MainContainer = styled.div`
  display: flex;
  justify-content: space-evenly;
  width: 100%;
  height: 100%;
  min-height: inherit;
`;

export const BarChartContainer = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: flex-end;
  align-items: center;
`;

export const Chart = css`
  margin-top: 0%;
  width: 100%;
  
  max-width: 96px;
  &:hover {
    opacity: 0.8;
  }
  @media (max-width: 420px) {
    width: 34px;
  }
`;

export const Number = styled.span`
  font-size: 1.5rem;
  text-align: center;
  color: ${(props) => props.color};
  background-image: ${(props)=>props.backgroundImage}
`;

export const MakeBar = styled.div`
  height: ${(props) => props.height}%;
  background-image: linear-gradient(
    to bottom,
    ${(props) => props.colors[0]},
    ${(props) => props.colors[1]}
  );
  ${Chart};
`;

export const BlackLine = styled.div`
  width: 100%;
  height: 5px;
  background-color: grey;
`;


   
  
const ImageChoice =(props)=> {
  const imgresult=props.imgresult
  

  
    return(
      <Container>
        <h1>{props.data2.question}</h1>
      <MainContainer>
        {imgresult.map((imgresult, i) => {
          return (
            <BarChartContainer key={i}>
             <div><img src={props.images.bgurl}/></div>
             <Number color={imgresult.colors[1]}  >{imgresult.distance}</Number>
              <MakeBar height={imgresult.distance * 2} colors={imgresult.colors} />
             
            
            </BarChartContainer>
          );
        })}
      </MainContainer>
      <BlackLine />
    </Container>
    );
  } 
  export default ImageChoice