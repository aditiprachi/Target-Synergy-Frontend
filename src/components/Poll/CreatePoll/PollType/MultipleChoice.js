import styled, { css } from "styled-components";
import React, {useState} from 'react'
import Button from '@material-ui/core/Button';


export const Container = styled.div`
  margin: 0px auto;
  max-width: 1000px;
  height: 100%;
  min-height: 400px;
  display: flex;
  flex-direction: column;
  justify-content: flex-end;
`;

export const MainContainer = styled.div`
  display: flex;
  justify-content: space-evenly;
  width: 100%;
  height: 100%;
  min-height: inherit;
`;

export const BarChartContainer = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: flex-end;
  align-items: center;
`;

export const Chart = css`
  margin-top: 0%;
  width: 100%;
  max-width: 86px;
  &:hover {
    opacity: 0.8;
  }
  @media (max-width: 420px) {
    width: 34px;
  }
`;

export const Number = styled.span`
  font-size: 1.5rem;
  text-align: center;
  color: ${(props) => props.color};
`;

export const MakeBar = styled.div`
  height: ${(props) => props.height}%;
  background-image: linear-gradient(
    to bottom,
    ${(props) => props.colors[0]},
    ${(props) => props.colors[1]}
  );
  ${Chart};
`;

export const BlackLine = styled.div`
  width: 100%;
  height: 5px;
  background-color: grey;
`;


  
  
function MultipleChoice ({data1, result, inputList, textcolor}) {
  
 

    return(
      <Container>
        <h2 style={{color: textcolor}}>{data1.question}</h2>
        <MainContainer>
        {result.map(( result, i) => {
          return (
            <BarChartContainer key={i}>
              <Number color={result.colors[1]}>{result.distance}</Number>
              <MakeBar height={result.distance * 2} colors={result.colors} />
              <b style={{color: textcolor}}>{inputList[i].choice}</b> 
              
            </BarChartContainer>
          );
        })}
      </MainContainer>
      <BlackLine />
    </Container>
    );
  } 
  export default MultipleChoice