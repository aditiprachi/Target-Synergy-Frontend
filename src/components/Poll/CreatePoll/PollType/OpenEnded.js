import styled, { css } from "styled-components";
import React,{useState,useEffect} from 'react'
import { makeStyles } from '@material-ui/core/styles';
import Box from '@material-ui/core/Box';
import randomColor from 'randomcolor'
import axios from 'axios'

const useStyles = makeStyles((theme) => ({
    root: {
        flex: 1,
      '& > *': {
        margin: theme.spacing(1),
        width: '100%',
      
      },
    },
    h:{
      fontSize: '15px',
      marginBottom: '0',

    },
    typography: {
        // for settings
        fontSize: 12,
      },
    button: {
        margin: theme.spacing(1),
        borderRadius: "2em",
        
      },
    addicon: {
        color: 'solid white',
    },
}));

export const Container = styled.div`
inline-size: 150px;
overflow-wrap: break-word;
text-align: center;
float: left;
  width: 250px;
  padding: 3px;
  margin: 3px;

`;


export const box = styled.div`
  height: 1%;

`;
const OpenEnded = (props) => {
    const [OpenEndedResponses , setOpenEndedResponses] = useState({responses:[]});
  console.log("hi")
   
    const classes = useStyles();
    
    return (
        <div >
            <h1 fontFamily= "Helvetica" justifyContent="left" >{props.openEnded.question}</h1>

       
    </div>
    );
}

export default OpenEnded
