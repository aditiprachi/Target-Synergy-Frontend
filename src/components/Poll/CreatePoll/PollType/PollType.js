import React from 'react'
import { makeStyles } from '@material-ui/core/styles';
import { ButtonGroup, Grid } from '@material-ui/core'
import EqualizerIcon from '@material-ui/icons/Equalizer';
import PhotoLibraryIcon from '@material-ui/icons/PhotoLibrary';
import AssignmentRoundedIcon from '@material-ui/icons/AssignmentRounded';
import CloudIcon from '@material-ui/icons/Cloud';
import TuneIcon from '@material-ui/icons/Tune';
import QuestionAnswerIcon from '@material-ui/icons/QuestionAnswer';
import { Button } from '@material-ui/core';



const useStyles = makeStyles((theme) => ({
    
    root: {
      flexGrow: 1,
    },
    togglebtn: {
      width: '110px',
      margin: '1px' ,
      marginRight: '5px',
      height: '100px',
      padding: theme.spacing(2),
      color: theme.palette.text.secondary,
      
    },
    icon: {
    
     display: 'block',
     
     
     
    
  },
  }));
const PollType = ({clickHandler, handleChangeIndex}) => {
    const classes = useStyles();
    const [alignment, setAlignment] = React.useState('mc');
   
    const handleAlignment = (event, newAlignment) => {
      setAlignment(newAlignment);
    };
    

 

        

    function FormRow() {
      
      
    
      return (
        
        <ButtonGroup
        
      value={alignment}
      exclusive
      onChange={handleAlignment}
      aria-label="text alignment"
      style={{marginBottom: '20px'}}
    >
          
            <Button value="mc" aria-label="multiplechoice" className={classes.togglebtn} onClick={()=>{clickHandler("multiplechoice"); handleChangeIndex(1)}}
          >
            <EqualizerIcon fontSize="large" className={classes.icon} /><h5>Multiple Choice</h5>
            </Button>
         
          <Button value="ic" aria-label="imagechoice" className={classes.togglebtn} onClick={()=>{clickHandler("imagechoice");handleChangeIndex(1)}}>
            <PhotoLibraryIcon fontSize="large" className={classes.icon}/><h5>Image Choice</h5>
            </Button>
            
            </ButtonGroup>
      );
    }

    
    function FormRow1() {
      return (
        
        <ButtonGroup
        style={{marginBottom: '20px'}}
      value={alignment}
      exclusive
      onChange={handleAlignment}
      aria-label="text alignment"
    >
          
            <Button value="oe" aria-label="open ended" className={classes.togglebtn} onClick={()=>{clickHandler("openended"); handleChangeIndex(1)}}>
            <AssignmentRoundedIcon fontSize="large" className={classes.icon} /><h5>Open Ended</h5>
            </Button>
  
         
            <Button value="wc" aria-label="word cloud" className={classes.togglebtn} onClick={()=>{clickHandler("wordcloud"); handleChangeIndex(1)}}  >
           <CloudIcon fontSize="large" className={classes.icon}/><h5>Word Cloud</h5>
            </Button>
          
          </ButtonGroup>
            );
    }

    function FormRow2() {
      return (
         
        <ButtonGroup
        style={{marginBottom: '20px'}}
      value={alignment}
      exclusive
      onChange={handleAlignment}
      aria-label="text alignment"
    >
          
            <Button value="sc" aria-label="scales" className={classes.togglebtn} onClick={()=>{clickHandler("scales");handleChangeIndex(1)}}>
       
            <TuneIcon fontSize="large" className={classes.icon} /><h5>Scales</h5>
           </Button>
         
          
           <Button value="qa" aria-label="Q and A" className={classes.togglebtn} onClick={()=>{clickHandler("qanda");handleChangeIndex(1)}}>
           <QuestionAnswerIcon fontSize="large" className={classes.icon}/><h5>Q and A</h5> 
           </Button>
            
            </ButtonGroup>
      );
    }
    return (
        <div className={classes.root}>
          <Grid container spacing={1}  ><h4>Questions Type</h4> 
            <Grid container item xs={12} spacing={3} alignItems="center">
              <FormRow clickHandler={clickHandler} />
            </Grid>
            <Grid container item xs={12} spacing={3} alignItems="center">
              <FormRow1 clickHandler={clickHandler}/>
            
            </Grid>
            <Grid container item xs={12} spacing={3} alignItems="center">
              <FormRow2 clickHandler={clickHandler}/>
            </Grid>
          </Grid>
        </div>
      );
}

export default PollType