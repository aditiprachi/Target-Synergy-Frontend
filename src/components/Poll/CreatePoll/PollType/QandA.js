import styled, { css } from "styled-components";
import React, {useState} from 'react'
import Button from '@material-ui/core/Button';


export const Container = styled.div`
  margin: 0px auto;
  max-width: 1000px;
  height: 100%;
  min-height: 400px;
  display: flex;
  flex-direction: column;
  justify-content: flex-end;
`;

export const MainContainer = styled.div`
  display: flex;
  justify-content: space-evenly;
  width: 100%;
  height: 100%;
  min-height: inherit;
`;

export const Chart = css`
  margin-top: 0%;
  width: 100%;
  max-width: 86px;
  &:hover {
    opacity: 0.8;
  }
  @media (max-width: 420px) {
    width: 34px;
  }
`;

export const Number = styled.span`
  font-size: 1.5rem;
  text-align: center;
  color: ${(props) => props.color};
`;

function QandA ({data6, result, textcolor}) {
    return(
      <Container>
        <h1 style={{color: textcolor}}>{data6.question}</h1>
        <MainContainer>
      </MainContainer>
    </Container>
    );
  } 
  export default QandA