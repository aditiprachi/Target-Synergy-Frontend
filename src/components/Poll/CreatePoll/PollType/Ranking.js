import React from 'react';
import Rating from '@material-ui/lab/Rating';
import Typography from '@material-ui/core/Typography';
import Box from '@material-ui/core/Box';

function Ranking({data2, inputList, textcolor}) {
  return (
    <div>
      <h1 style={{color: textcolor}}>{data2.question}</h1>
      {inputList.map((x,i)=>{
            return(
              <Box component="fieldset" mb={3} borderColor="transparent" key={i}>
                <Typography style={{color: textcolor}}>{inputList[i].option}</Typography> 
                <Rating
                defaultValue={0}
                precision={0.5}
                max={5}
                size="large"
                />
              </Box>  )
        })}
    </div>
  );
}

export default Ranking