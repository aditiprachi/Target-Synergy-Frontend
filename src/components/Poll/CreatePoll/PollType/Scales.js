import React from 'react';
import PropTypes from 'prop-types';
import { withStyles, makeStyles } from '@material-ui/core/styles';
import Slider from '@material-ui/core/Slider';
import { Container } from '@material-ui/core';

const Scales =({data5,inputList}) => {
    const ScaleSlider = withStyles({
        root: {
          color: '#52af77',
          height: 8,
        },
        thumb: {
          height: 24,
          width: 24,
          backgroundColor: '#fff',
          border: '2px solid currentColor',
          marginTop: -8,
          marginLeft: -12,
          '&:focus, &:hover, &$active': {
            boxShadow: 'inherit',
          },
        },
        active: {},
        valueLabel: {
          left: 'calc(-50% + 4px)',
        },
        track: {
          height: 8,
          borderRadius: 4,
        },
        rail: {
          height: 8,
          borderRadius: 4,
        },
      })(Slider);
    return(
        <div>
            <h1>{data5.question}</h1>
        {inputList.map((x,i)=>{
            return(
            <Container key={i}>
        <b style={{textAlign: 'left'}}>{inputList[i].choice}</b>  
     <ScaleSlider  defaultValue={0}  />
</Container>)
        })}
        </div>
    );
};
export default Scales