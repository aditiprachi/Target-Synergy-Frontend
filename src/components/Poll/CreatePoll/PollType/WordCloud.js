import React,{useState, useEffect,useContext} from 'react';
import ReactWordcloud from 'react-wordcloud';
import axios from 'axios'
import { IdContext } from '../../../../IdContext';


const wcr=[];
const WordCloud=({data4,textcolor})=>{
  const id = useContext(IdContext);
  const url = id.id;

    useEffect(async () => {
      var result = await axios.get(`http://localhost:8080${url}`)
      console.log(result.data)

    let i=0
    Object.entries(result.data).forEach(([key, value]) => {
  result.value = true;
  wcr[i]=({text:`${key}`, value:`${value}`})
  i++
})
console.log(wcr)
},[]);
let wcrs=[...wcr]
console.log(wcrs)

const resizeStyle = {
    display: "flex",
    alignItems: "center",
    justifyContent: "center",
    border: "solid 1px #ddd",
    width: '100%',
    height: '90%',
    marginTop: '5%'
  };
  const callbacks = {
    getWordColor: word => word.value > 50 ? "blue" : "red",
    onWordClick: console.log,
    onWordMouseOver: console.log,
    getWordTooltip: word => `${word.text} (${word.value}) [${word.value > 50 ? "good" : "bad"}]`,
  }
 
  const options = {
    colors: ["#1f77b4", "#ff7f0e", "#2ca02c", "#d62728", "#9467bd", "#8c564b"],
    enableTooltip: true,
    deterministic: false,
    fontFamily: "helvetica",
    fontSizes: [ 30, 100],
    fontStyle: "normal",
    fontWeight: "normal",
    padding: 1,
    rotations: 2,
    rotationAngles: [0, 90, -90],
    scale: "sqrt",
    spiral: "archimedean",
    transitionDuration: 1000
  };
  const size = [1200, 400];

    return (
      <div>
        <h1 style={{color: textcolor}}>{data4.question}</h1>
        <div style={resizeStyle}>
      <ReactWordcloud
        callbacks={callbacks}
        options={options}
        size={size}
        words={wcr}
      />
      </div>
      </div>
    );
  }
  export default WordCloud;