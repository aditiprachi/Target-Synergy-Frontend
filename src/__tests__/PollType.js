import React from "react";
import PollType from "../components/Poll/CreatePoll/PollType/PollType";
import renderer from "react-test-renderer";

describe("Testing Snapshot for PollType.js",()=>{
    it("Renders",()=>{
        const tree = renderer.create(<PollType clickHandler={[""]} handleChangeIndex={[""]} />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})