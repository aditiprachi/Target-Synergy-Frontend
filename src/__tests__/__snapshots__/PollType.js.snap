// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Snapshot for PollType.js Renders 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
  >
    <h4>
      Questions Type
    </h4>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-grid-xs-12"
    >
      <div
        aria-label="text alignment"
        className="MuiButtonGroup-root"
        exclusive={true}
        onChange={[Function]}
        role="group"
        style={
          Object {
            "marginBottom": "20px",
          }
        }
        value="mc"
      >
        <button
          aria-label="multiplechoice"
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-togglebtn-2"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
          value="mc"
        >
          <span
            className="MuiButton-label"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root makeStyles-icon-3 MuiSvgIcon-fontSizeLarge"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 20h4V4h-4v16zm-6 0h4v-8H4v8zM16 9v11h4V9h-4z"
              />
            </svg>
            <h5>
              Multiple Choice
            </h5>
          </span>
        </button>
        <button
          aria-label="rating"
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-togglebtn-2"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
          value="ic"
        >
          <span
            className="MuiButton-label"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root makeStyles-icon-3 MuiSvgIcon-fontSizeLarge"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
              />
            </svg>
            <h5>
              RATING
            </h5>
          </span>
        </button>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-grid-xs-12"
    >
      <div
        aria-label="text alignment"
        className="MuiButtonGroup-root"
        exclusive={true}
        onChange={[Function]}
        role="group"
        style={
          Object {
            "marginBottom": "20px",
          }
        }
        value="mc"
      >
        <button
          aria-label="open ended"
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-togglebtn-2"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
          value="oe"
        >
          <span
            className="MuiButton-label"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root makeStyles-icon-3 MuiSvgIcon-fontSizeLarge"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 3h-4.18C14.4 1.84 13.3 1 12 1s-2.4.84-2.82 2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-7 0c.55 0 1 .45 1 1s-.45 1-1 1-1-.45-1-1 .45-1 1-1zm1 14H8c-.55 0-1-.45-1-1s.45-1 1-1h5c.55 0 1 .45 1 1s-.45 1-1 1zm3-4H8c-.55 0-1-.45-1-1s.45-1 1-1h8c.55 0 1 .45 1 1s-.45 1-1 1zm0-4H8c-.55 0-1-.45-1-1s.45-1 1-1h8c.55 0 1 .45 1 1s-.45 1-1 1z"
              />
            </svg>
            <h5>
              Open Ended
            </h5>
          </span>
        </button>
        <button
          aria-label="word cloud"
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-togglebtn-2"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
          value="wc"
        >
          <span
            className="MuiButton-label"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root makeStyles-icon-3 MuiSvgIcon-fontSizeLarge"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19.35 10.04C18.67 6.59 15.64 4 12 4 9.11 4 6.6 5.64 5.35 8.04 2.34 8.36 0 10.91 0 14c0 3.31 2.69 6 6 6h13c2.76 0 5-2.24 5-5 0-2.64-2.05-4.78-4.65-4.96z"
              />
            </svg>
            <h5>
              Word Cloud
            </h5>
          </span>
        </button>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-grid-xs-12"
    >
      <div
        aria-label="text alignment"
        className="MuiButtonGroup-root"
        exclusive={true}
        onChange={[Function]}
        role="group"
        style={
          Object {
            "marginBottom": "20px",
          }
        }
        value="mc"
      >
        <button
          aria-label="scales"
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-togglebtn-2"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
          value="sc"
        >
          <span
            className="MuiButton-label"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root makeStyles-icon-3 MuiSvgIcon-fontSizeLarge"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 17v2h6v-2H3zM3 5v2h10V5H3zm10 16v-2h8v-2h-8v-2h-2v6h2zM7 9v2H3v2h4v2h2V9H7zm14 4v-2H11v2h10zm-6-4h2V7h4V5h-4V3h-2v6z"
              />
            </svg>
            <h5>
              Scales
            </h5>
          </span>
        </button>
        <button
          aria-label="Q and A"
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedOutlined MuiButtonGroup-groupedOutlinedHorizontal MuiButtonGroup-groupedOutlined makeStyles-togglebtn-2"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
          value="qa"
        >
          <span
            className="MuiButton-label"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root makeStyles-icon-3 MuiSvgIcon-fontSizeLarge"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M21 6h-2v9H6v2c0 .55.45 1 1 1h11l4 4V7c0-.55-.45-1-1-1zm-4 6V3c0-.55-.45-1-1-1H3c-.55 0-1 .45-1 1v14l4-4h10c.55 0 1-.45 1-1z"
              />
            </svg>
            <h5>
              Q and A
            </h5>
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;
